tests/jballerina-unit-test/src/test/resources/test-src/bir/biroptimizer.bal

```diff
@@ -55,10 +55,9 @@ public function globalVarsAndAnonFunctions() {
     };
 }
 
-service hello on new http:Listener(9090) {
-    resource function sayHello(http:Caller caller,
-        http:Request req) returns error? {
-        check caller->respond("Hello, World!");
+service hello on new test:MockListener(9090) {
+    resource function sayHello() returns error? {
+
```

Comment:
Shall we mock the `client object Caller` as well. Since this test is about BIR instructions I think we can't leave the function body empty.

Reply:
Done.